cmake/Modules/Findgtest.cmake

```diff
@@ -6,8 +6,8 @@ find_package(Threads REQUIRED)
 include(ExternalProject)
 ExternalProject_Add(
     gtest
-    URL http://github.com/google/googletest/archive/release-1.7.0.zip
-    URL_MD5 ef5e700c8a0f3ee123e2e0209b8b4961
+    # we provide our own command here, since default curl binary used by Cmake might not support SSL.
```

Comment:
I would add a comment that we can revert this to the old style once we get off centos 5 -- the other method is much better.